entrypoint.sh

```diff
@@ -31,16 +32,20 @@ if [ "$1" = "kubelet" ]; then
     done
     mount --rbind /host/dev /dev
     mount -o rw,remount /sys/fs/cgroup 2>/dev/null || true
-    for i in /sys/fs/cgroup/*; do
+
+    # Only applicable to cgroup v1
```

Comment:
if this is only applicable to cgroup v1, do you also need to query `.Info.CgroupVersion` from the docker info?

Reply:
Related to my previous comment, the `-f` option is not available with the docker binary shipped in the image.